skycoin/skywire
# Use default values if required visor config fields are empty

Did you run `make format && make check`? Yes

Fixes #168
Depends on https://github.com/SkycoinPro/skywire-services/pull/127

 Changes:	
- Use default values if required visor config fields are empty
- Cleanup visor config and related configs

How to test this PR:
- `make test`
- Run integration env, then check if nothing is broken
- Remove visor keys from integration env configs, run integration env. Scripts for checking communication should be broken. Visor logs should contain messages `INFO []: Starting visor PK=` with generated PKs instead of empty PKs. Restart integration env and check if generated PKs changed.
- Generate configs with `skywire-cli`:
  - `./skywire-cli visor gen-config -o ./test.json -r -m WD`
  - `./skywire-cli visor gen-config -o ./test.json -r -m HOME`
  - `./skywire-cli visor gen-config -o ./test.json -r -m LOCAL`
